scalafix-cli/src/main/scala/scalafix/internal/cli/ClasspathOps.scala

```diff
@@ -1,8 +1,41 @@
 package scalafix.internal.cli
 
 import java.io.File
+import java.io.PrintStream
+import scala.meta.Classpath
+import scala.meta.metacp
+import scalafix.internal.util.LazySymbolTable
+import scalafix.internal.util.SymbolTable
 
 object ClasspathOps {
+
+  def bootClasspath: Option[Classpath] = sys.props.collectFirst {
+    case (k, v) if k.endsWith(".boot.class.path") => Classpath(v)
+  }
+
+  /**
+    * Process classpath with metacp to build semanticdbs of global symbols.
+    *
+    * @param sclasspath Regular classpath to process.
+    * @param out The output stream to print out error messages.
+    */
+  def toMetaClasspath(sclasspath: Classpath, out: PrintStream): Classpath = {
```

Comment:
The fact that these methods take a printstream looks really awkward.

Reply:
Use System.out directly.

Reply:
Scalafix runs via nailgun, in IDEs with custom logging, etc, 

It's not OK to use System.out